test/integration/serverless/snapshots/log-java

```diff
@@ -3,27 +3,29 @@
     "message": {
       "message": "Picked up JAVA_TOOL_OPTIONS: -javaagent:\"/opt/java/lib/dd-java-agent.jar\" -XX:+TieredCompilation -XX:TieredStopAtLevel=1\n",
       "lambda": {
-        "arn": ""
+        "arn": "arn:aws:lambda:eu-west-1:601427279990:function:integration-tests-extension-STAGE-log-java",
+        "request_id": "XXX"
       }
     },
     "status": "info",
-    "hostname": "",
+    "hostname": "arn:aws:lambda:eu-west-1:601427279990:function:integration-tests-extension-STAGE-log-java",
```

Comment:
Correct me if I'm wrong, but isn't it true we never expect to receive a hostname value? Or am I confusing it with a `host` tag?